### DISCUSSION TITLE
How to create a flatpak program for briefcase?

### DISCUSSION THREAD

**Discussion by yjlc-pc** [beeware/briefcase#1739] [Q&A] [answered]
I can find "create flatpak program" option for https://briefcase.readthedocs.io/en/latest/reference/platforms/linux/flatpak.html ,but I can't find create flatpak program's command.

**Comment by mhsmith** [comment, marked as answer]
The commands are:

* `briefcase create linux flatpak`
* `briefcase build linux flatpak`

For full details of the `create` and `build` commands, see the [documentation](https://briefcase.readthedocs.io/en/latest/reference/commands/index.html).
